syntax = "proto3";

message HeartbeatACK {
    int32 random = 1;
}